mathics/builtin/arithmetic.py

```diff
@@ -208,6 +208,13 @@ class Abs(_MPMathFunction):
     summary_text = "absolute value of a number"
     sympy_name = "Abs"
 
+    def eval(self, x, evaluation):
+        "%(name)s[x_]"
+        result = eval_abs(x)
```

Comment:
Should follow convention using capital letter. Also it would be nice to start removing `%(name)s`. 

I will be putting in a PR to merge into this to correct such things.